api/v1alpha3/awscluster_types.go

```diff
@@ -146,9 +146,9 @@ type Bastion struct {
 
 // AWSLoadBalancerSpec defines the desired state of an AWS load balancer.
 type AWSLoadBalancerSpec struct {
-	// Scheme sets the scheme of the load balancer (defaults to internet-facing)
-	// +kubebuilder:default=internet-facing
-	// +kubebuilder:validation:Enum=internet-facing;internal
+	// Scheme sets the scheme of the load balancer (defaults to Internet-facing)
+	// +kubebuilder:default=Internet-facing
+	// +kubebuilder:validation:Enum=Internet-facing;internal
```

Comment:
```suggestion
	// +kubebuilder:default=internet-facing
	// +kubebuilder:validation:Enum=internet-facing;Internet-facing;internal
```

Or have I misunderstood the problem entirely?